src/components/hero-section/heroSection.tsx

```diff
@@ -1,55 +1,115 @@
-      <div className={styles.mainBlock}>
-        <div className={styles.slogan}>
-          <h1 className={styles.sloganHeader}>
+    <Box component="section">
+      <Container
+        sx={{
+          ...mainBlock,
+          [theme.breakpoints.between('sm', 'lg')]: mainBlock.lg,
+          [theme.breakpoints.down('sm')]: mainBlock.sm
+        }}
+      >
+        <Box sx={slogan}>
+          <Typography
+            sx={{
+              color: 'secondary.darker',
+              [theme.breakpoints.between('sm', 'lg')]: sloganTitle.md,
+              [theme.breakpoints.down('sm')]: sloganTitle.sm
+            }}
+            component="h1"
+            variant="h1"
+          >
             Найсолодші ласощі <br />
-            <span className={styles.sloganHeaderItalic}> від природи, </span>
+            <em>від природи,</em>
             <br /> зроблені з любов`ю
-          </h1>
-          <p className={styles.sloganParagraph}>
+          </Typography>
+          <Typography
+            component="p"
+            variant="body1"
+            sx={{
+              ...sloganParagraph
+            }}
+          >
             Відкрийте для себе неперевершені смаки нашої <br /> повністю
             натуральної фруктової <br /> пастили та фріпсів
-          </p>
+          </Typography>
           <Link
             style={{ width: '100%', maxWidth: '300px' }}
-            to={`/catalog/${allCategories && allCategories[0].id}`}
+            to={firstCategoryId}
           >
             <Button
               fullWidth
-              sx={{
-                borderRadius: '20px',
-                backgroundColor: theme.palette.primary.darker,
-                color: '#fff',
-                height: '60px'
-              }}
+              variant="contained"
+              sx={{ [theme.breakpoints.down('md')]: sloganButton }}
             >
               Переглянути каталог
             </Button>
           </Link>
-        </div>
-        <img
-          src={HeroPic}
-          alt="dried fruits in plastic bags"
-          width={760}
-          height={730}
-        />
-      </div>
-    </section>
+        </Box>
+
+        <Box
+          sx={{
+            [theme.breakpoints.down('sm')]: {
+              height: 'clamp(14.375rem, 4.375rem + 50vw, 23.125rem)'
+            }
+          }}
+        >
+          {isSmallScreen ? (
+            <Box
+              component="img"
+              src={HeroMobilePic}
+              alt="dried fruits in plastic bags"
+              sx={{
+                ...sloganImgLogo.sm
+              }}
+            />
+          ) : (
+            <Box
+              height={205}
+              component="img"
+              src={HeroPic}
+              alt="dried fruits in plastic bags"
+              sx={{
+                ...sloganImgLogo.lg,
+                [theme.breakpoints.between('sm', 'md')]: sloganImgLogo.md
+              }}
+            />
+          )}
```

Comment:
you can do it in ease way, something like this

```suggestion
          <Box
          height={isSmallScreen ? undefined : 205}
          component="img"
          src={isSmallScreen ? HeroMobilePic : HeroPic}
          alt="dried fruits in plastic bags"
          sx={isSmallScreen ? sloganImgLogo.sm : {...sloganImgLogo.lg, [theme.breakpoints.between('sm', 'md')]: sloganImgLogo.md}}
        />
```

Reply:
Done